WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Nothing shall be blocked?

"Blocking" would be to just ignore your changes.

> Then why are you dragging this PR on for months on petty issues? I reverted a butt ton of my code at your behest, and you still give me no slack.

There is an ongoing review of this content. As I have said, no half-read code will be included, everything will be verified and reviewed thoroughly. And how is this petty all of a sudden? You yourself has said that this "was the most broken file", that the code here "is the worst", that it is "completely terrible" and that you are personally "desperate". That indicates you seem to believe this is major. Re-read your own posts and explain to me how you changed your mind. 

> I've explained my desired changes to you over and over again. It's not my fault if you don't have the means to understand, by now, what I want to do. You've already seen the code in the Mercenaries file, and you have no legitimate reasons for not wanting those changes to be accepted.

You haven't done this at all. All you have signalled is that you want to rush through this for the sake of it. You scream about your anger and vent discontent, but you refuse to open an issue dedicated to this explaining your view. This isn't some sort of protest forum, reconsider your method here.

> I can go elsewhere and get someone to do it if you aren't willing to listen to me.

If by elsewhere you mean open an issue, yes, do so. Someone else can open the issue too and explain these changes.